membersuite_api_client/tests/test_memberships.py

```diff
@@ -21,6 +24,14 @@ def test_get_membership_for_org(self):
                          '6faf90e4-0074-cbb5-c1d2-0b3c539859ef')
 
         # Test org without a membership
+
+        # Need an Organization ID for a non-member org. The one here
+        # is the same used above for testing with a member org.
+        #
+        #            HERE!
+        #              |
+        #              |
+        #              V
         test_org_id = "6faf90e4-0007-c9dc-98b7-0b3c53985743"
```

Comment:
Line 36 tries querying memberships for "6faf90e4-0007-c9dc-98b7-0b3c53985743", but should return an empty list.

I think MemberSuite's ID system is confusing, here. IDs for all objects have the same format. 

"6faf90e4-0007-c578-8310-0b3c53985743" is an organization that does have an associated membership.
'6faf90e4-0074-cbb5-c1d2-0b3c539859ef' is that associated membership.
"6faf90e4-0007-c9dc-98b7-0b3c53985743" is another organization that does NOT have a memebrship, so the lines below it test that calling "get_memberships_for_org()" returns an empty result list.